fluxc/src/main/java/org/wordpress/android/fluxc/store/PostStore.java

```diff
@@ -695,6 +696,19 @@ private void handleFetchPostsCompleted(FetchPostsResponsePayload payload) {
         emitChange(onPostChanged);
     }
 
+    private void handleFetchSinglePostCheckLocallyChanged(FetchPostResponsePayload payload) {
+        // Process a fetch action only for a post if it hasn't local changes
```

Comment:
Nitpick: There is a small grammatical mistake in this sentence.

```suggestion
        // Process a fetch action for a post only if it doesn't have local changes
```